tools/wasm2c.py

```diff
@@ -207,6 +207,29 @@ def name(i):
   else:
     exit_with_error('bad sandboxing')
 
+  # adjust prefixing: emit simple output that works with multiple libraries,
+  # each compiled into its own single .c file, by adding 'static' in some places
+  # TODO: decide on the proper pattern for this in an upstream discussion in
+  #       wasm2c; another option would be to prefix all these things.
+  for rep in [
+    'uint32_t wasm_rt_register_func_type(',
+    'void wasm_rt_trap(',
+    'void wasm_rt_allocate_memory(',
+    'uint32_t wasm_rt_grow_memory(',
+    'void wasm_rt_allocate_table(',
+    'jmp_buf g_jmp_buf',
+    'uint32_t g_func_type_count',
+    'FuncType* g_func_types',
+    'uint32_t wasm_rt_call_stack_depth',
+    'uint32_t g_saved_call_stack_depth',
+  ]:
+    # remove 'extern' from declaration
+    total = total.replace('extern ' + rep, rep)
+    # add 'static' to implementation
+    old = total
+    total = total.replace(rep, 'static ' + rep)
+    assert old != total, f'did not find "{rep}"'
```

Comment:
This seems a bit hacky to just use search/replace here but as you say we can do better with future wabt changes maybe.

Reply:
Yes, agreed it's hacky. But this at least gets us from totally broken to hacky :wink: (and the proper non-hacky solution may require lengthy discussion, which I'd rather not wait for as multiple users would like this).